diff --git a/dist/lib/typescript/writeConfigurationDefaults.js b/dist/lib/typescript/writeConfigurationDefaults.js
index de155863ae840882bb52c1537196f87cdc8c714f..f8156e983275e5d81ac8447cc8e8e66e058e7589 100644
--- a/dist/lib/typescript/writeConfigurationDefaults.js
+++ b/dist/lib/typescript/writeConfigurationDefaults.js
@@ -229,7 +229,8 @@ async function writeConfigurationDefaults(ts, tsConfigPath, isFirstTimeSetup, is
     if (suggestedActions.length < 1 && requiredActions.length < 1) {
         return;
     }
-    await _fs.promises.writeFile(tsConfigPath, CommentJson.stringify(userTsConfig, null, 2) + _os.default.EOL);
+    console.log('patch prevented next from writing tsconfig.json');
+    // await _fs.promises.writeFile(tsConfigPath, CommentJson.stringify(userTsConfig, null, 2) + _os.default.EOL);
     if (isFirstTimeSetup) {
         console.log(_chalk.default.green(`We detected TypeScript in your project and created a ${_chalk.default.bold("tsconfig.json")} file for you.`) + "\n");
         return;
